### PR TITLE
typo and broken link in README

### PR DESCRIPTION
**Pull request by andyczerwonka**
As I go through the documentation I found a couple of typo's, a broken link, etc. Wondering it you would like me to capture this in a PR as I go through it?

**Highlights**
* a couple of typos
* broken link to the `Measurement Run`
* some very subjective cosmetic changes that hopefully don't change the semantics

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -1,27 +1,21 @@
...
+The model links together materials, the processes that produced them, and the measurements that characterize them.
+This facilitates the backwards traversal from a measurement to the material on which it was performed to the process by which it was produced to the materials which were used in that process. 
+It generalizes and matures `preparation` and `subSystems` objects within the Physical Information File (PIF).
 
-Additionally, this format makes a first-class distinction between *intent* and *realization*, 
-captured by `Spec` and `Run` objects, respectively.
-A single intent `Spec` can be realized into multiple `Run` objects.
-This generalized and matures the `ideal` concept from the PIF's `Composition` and `Quantity` objects.
+Additionally, the model makes a first-class distinction between *intent* and *realization*, captured by `Spec` and `Run` objects, respectively. 
+A single intent `Spec` can be realized into multiple `Run` objects. This generalizes and matures the `ideal` concept from the PIF's `Composition` and `Quantity` objects.
```
Comment (maxhutch) [inline docs/README.md]:
```suggestion
A single intent `Spec` can be realized into multiple `Run` objects.
This generalizes and matures the `ideal` concept from the PIF's `Composition` and `Quantity` objects.
```
maintain (no more than) one sentence per line

Comment (andyczerwonka) [reply docs/README.md]: Great! To keep it manageable, I'll remove the DO NOT MERGE and issue new PR's as I go through all of this documentation in the next week or so.

**Comment by andyczerwonka** [comment]
Great! To keep it manageable, I've removed the DO NOT MERGE and will open new PR's as I go through all of this documentation in the next week or so.

File: docs/README.md [hunk 2]
```diff
@@ -1,27 +1,22 @@
...
+This is the next major data model developed by Citrine, codename `taurus`.
 
-The format links together materials, the processes that produced them,
-and the measurements that characterize them.
-This facilitates the backwards traversal from a measurement to the material on which it was performed
-to the process by which it was produced to the materials which were used in that process and so on and so forth.
-This generalizes and matures the PIF's `preparation` and `subSystems` objects.
+The model links together materials, the processes that produced them, and the measurements that characterize them.
+This facilitates the backwards traversal from a measurement to the material on which it was performed to the process by which it was produced to the materials which were used in that process. 
+It generalizes and matures `preparation` and `subSystems` objects within the Physical Information File (PIF).
```
Comment (kroenlein) [inline docs/README.md]:
```suggestion
It generalizes and matures `preparation` and `subSystems` objects within the PIF ([Physical Information File](https://citrineinformatics.github.io/pif-documentation/)).
```

As the document is more likely read by someone familiar with our previous technology, I think the acronym should precede.

**Pull request merged**